dcos/mesos.py

```diff
@@ -30,14 +30,8 @@ class DCOSClient(object):
 
     def __init__(self):
         config = util.get_config()
-        self._dcos_url = None
-        self._mesos_master_url = None
-
-        mesos_master_url = config.get('core.mesos_master_url')
-        if mesos_master_url is None:
-            self._dcos_url = util.get_config_vals(['core.dcos_url'], config)[0]
-        else:
-            self._mesos_master_url = mesos_master_url
+        self._dcos_url = util.get_config_vals(['core.dcos_url'], config)[0]
```

Comment:
I considered doing this, but I think the call to get_config_vals acts a check to make sure the variable is provided. 

I think a better solution would be to remove all the checks for `self._dcos_url` in DCOSClient and leave the `util.get_config_vals` in the to confirm its existence. Sounds good? (Making changes as I post this)